app.rb

```diff
@@ -0,0 +1,98 @@
+
+module Enumerable
+  def my_each(&prc)
+    length.times do |i|
+      prc.call(self[i])
+    end
+    self
+  end
```

Comment:
- [ ] `my_each` is not working as the original `each` method, also is not passing all test cases.

### `each` is an Enumerator that has these characteristics: 
- it applies the block for each element inside the array/range/hash. Yours fails with hashes and ranges

```ruby
[1, 2, 3].each { |num| puts num }
# 1
# 2
# 3
# => [1, 2, 3]
```
[Enumerator#each](https://ruby-doc.org/core-2.7.2/Enumerator.html#method-i-each)